app/ui/src/app/common/ui-patternfly/notification-service.ts

```diff
@@ -39,4 +39,17 @@ export class NotificationService extends NS {
   getNotificationsObservable(): Observable<Notification[]> {
     return this.notificationsSubject.asObservable();
   }
+
+  // Toast notification wrapper, used for notifications
+  // that are not persistent and do not have actions associated
+  popNotification(notification) {
+    this.message(
+      notification.type,
+      notification.header,
+      notification.message,
+      false,
+      null,
+      []
+    );
```

Comment:
I've noticed that the `NotificationService.message()` is also called from other instances of the application with a similar payload in the signature. So now that we're updating the service, perhaps we can also simplify calls to the `message()` method by introducing default parameter values like this:


```typescript
message(
    type: string,
    header: string,
    message: string,
    isPersistent = false, // <--- This
    primaryAction?: Action, // <--- This
    moreActions: Action[] = [] // <--- This
  ): void {
    super.message(
      type,
      header,
      message,
      isPersistent,
      primaryAction,
      moreActions
    );
    this.notificationsSubject.next(this.getNotifications());
  }
```

This would also allow for a more leaner approach when calling the method by means of ES6 deconstruction:

```typescript
popNotification(notification): void {
  const { type, header, message } = notification;
  this.message(type, header, message);
}
```

Also please note that now we'll annotate the return type in the code example. Make a habit of annotating methods with the returned type, or 'void' if none, so we can better our code into a self-documenting experience.
Leaner, simpler, hipster code, yay

Reply:
Actually, the above can be simplified even more, at the cost of replacing strongly typed payloads by type inference, like this:

```typescript
popNotification({ type, header, message }): void {
  this.message(type, header, message);
}
```